File: python/questao3.py
def somatorio_n_primeiros(n):
    soma = sum(range(1, n + 1))
    return soma

n = int(input("Digite um número inteiro N: "))
soma = somatorio_n_primeiros(n)
print(f"Somatório dos primeiros {n} números: {soma}")

File: c/questao3.c
#include <stdio.h>

int somatorioNPrimeiros(int n) {
    int soma = 0;
    for (int i = 1; i <= n; i++) {
        soma += i;
    }
    return soma;
}

int main() {
    int n;

    printf("Digite um número inteiro N: ");
    scanf("%d", &n);

    int soma = somatorioNPrimeiros(n);
    printf("Somatório dos primeiros %d números: %d\n", n, soma);
    return 0;
}
